geode-core/src/main/java/org/apache/geode/management/internal/util/ClasspathScanLoadHelper.java

```diff
@@ -41,6 +41,12 @@ public ClasspathScanLoadHelper(Collection<String> packagesToScan) {
         .enableAnnotationInfo().scan(1);
```

Comment:
have this call the other constructor instead `this(packagesToScan.toArray(new String[] {}))`